arch/risc-v/src/common/riscv_sigdeliver.c

```diff
@@ -167,5 +167,5 @@ void riscv_sigdeliver(void)
   g_running_tasks[this_cpu()] = NULL;
```

Comment:
This will now cause a crash, you must ensure this patch is merged only after the other patch which fixes this is merged.

Reply:
It might cause conflicts, but it won't crash.